test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -193,33 +232,50 @@ private void basicSaiTest(EntropySource rng, SchemaSpec schema)
             if (rng.nextFloat() > 0.9995f)
                 history.deletePartition(partitionIndex);
 
-            if (i % REPAIR_SKIP == 0)
-                history.custom(() -> repair(schema), "Repair");
-            else if (i % FLUSH_SKIP == 0)
+            if (i % FLUSH_SKIP == 0)
                 history.custom(() -> flush(schema), "Flush");
             else if (i % COMPACTION_SKIP == 0)
                 history.custom(() -> compact(schema), "Compact");
+            else if (i % repairSkip == 0)
+                history.custom(() -> repair(schema), "Repair");
 
-            if (i > 0 && i % 1000 == 0)
+            if (i > 0 && i % VALIDATION_SKIP == 0)
```

Comment:
what motivated the change from `1k` to `739`?

Reply:
Just wanted it to be a little more frequent and offset from repair/flush/compaction